thirdeye/thirdeye-frontend/app/pods/components/events-table/component.js

```diff
@@ -52,57 +52,62 @@ export default Component.extend({
   ),
 
   // Require to display a loader for long rendering
-  didUpdateAttrs(...args) {
+  didUpdateAttrs() {
     later(() => {
-      this._super(args);
+      this._super(...arguments);
     });
   },
 
-  /**
-   * Columns required by ember-models-table
-   */
-  columns: [
```

Comment:
You can use Object.freeze(...)

Reply:
oh thanks! didn't know about this work around